src/app/find/mapPage/filters/filters.css

```diff
@@ -29,3 +29,13 @@ input#hivNutrition-checkbox {
 	margin-left: 32px;
 	color: #333333;
 }
+
+.filterSubmitBtn {
+    position: fixed;
```

Comment:
Indentation is inconsistent with the rest of the file.

Reply:
The selector above was using tabs, fixed it to match the others.